src/finance-page/finance-page.css

```diff
@@ -0,0 +1,19 @@
+.full-width {
+    margin: 0 auto;
+    width: 100%;
+}
+
+.finance-page {
+    position: relative;
+}
+
+.page-content {
+    position: fixed;
```

Comment:
please don't make page-content `position: fixed`. It may fix some issues for you now, but this won't work in a lot of cases and is bad practice.